### PR TITLE
Activity fix bug

### PR DESCRIPTION
**Pull request by ivyyangyq**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -432,7 +432,7 @@ the bottom of the list.
 Example:
 
 * `activityAdd n/Sentosa Outing d/16/02/2019 1000 l/Harbour Front MRT de/Bring swim suits.`
-* `activityAdd n/HTML workshop d/1402019 l/ICube Auditorium ic/A0123456Z`
+* `activityAdd n/HTML workshop d/14/02/2019 l/ICube Auditorium ic/A0123456Z`
```
Comment (minernchan) [inline docs/UserGuide.adoc]: the ic field no longer applicable right?

Comment (ivyyangyq) [reply docs/UserGuide.adoc]: ok i'll remove it

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -633,7 +633,7 @@ e.g. `activityDelete 1`
 e.g. `activityDeleteMember 1 A1234567N`
 * *Edit an activity* : `activityEdit ACTIVITY_INDEX [n/ACTIVITY_NAME] [d/DATE] [t/TAG] … ` +
 e.g. `activityEdit 1 d/13022019`
-* *Find an activity* :`activityFind [PREFIX] KEYWORD [MORE_KEYWORDS] [NEXT_PREFIX KEYWORD] [MORE_KEYWORDS]` +
+* *Find an activity* : `activityFind [PREFIX] KEYWORD [MORE_KEYWORDS] [NEXT_PREFIX KEYWORD] [MORE_KEYWORDS]` +
 e.g. `activityFind n/ Workshop`
 * *Filtering activities* : `activityFilter CONDITION NUMBER_OF_DAYS` +
 e.g. `activityFilter + 30`
```
Comment (A0143487X-Kevin) [inline docs/UserGuide.adoc]: Saw your UG on the activityFilter function
activityFilter + 30 means from now to 30 days later inclusive
what abt the - condition? maybe can also give an example

**Comment by A0143487X-Kevin** [comment]
UserGuide.adoc
Saw your UG on the activityFilter function
activityFilter + 30 means from now to 30 days later inclusive
what abt the - condition? maybe can also give an example

ActivityDateTime.java
Within isValidTime(String time)
return !(hour > 23 || minute > 59);
to be more accurate you might want to check (though it shouldnt affect functionality)
00-23 and 00-59

**Comment by A0143487X-Kevin** [comment]
Will merge for now cos omitting my suggestions should not affect functionality

**Pull request merged**